adapters/sleep.go

```diff
@@ -0,0 +1,28 @@
+package adapters
+
+import (
+	"github.com/smartcontractkit/chainlink/store"
+	"github.com/smartcontractkit/chainlink/store/models"
+)
+
+// Sleep adapter allows a job to do nothing for some amount of wall time.
+type Sleep struct {
+	EndAt models.Time `json:"until"`
```

Comment:
Should we consolidate the attribute name `EndAt` and the json key `until`? Only use `Until` for consistency? or vice versa